modules/lvgl/Kconfig

```diff
@@ -127,6 +127,12 @@ config LV_Z_AREA_Y_ALIGNMENT_WIDTH
 	  the current frame dimensions to meet display and/or LCD host
 	  controller requirements. The value must be power of 2.
 
+config LV_USE_GPU_STM32_DMA2D
+	bool
+
+config LV_GPU_DMA2D_CMSIS_INCLUDE
+	string
```

Comment:
Could be useful to add help section.
First symbol is explicit but for second one it might be usefull to explain if this expects a path, a file , ....

Reply:
Added the help description from LVGL.